k-distribution/pl-tutorial/1_k/4_imp++/lesson_3/README.md

```diff
@@ -31,99 +31,6 @@ the `krun` tool with the option `--search`. For example:
 
     krun div.imp --search
 
-Oops, we see only one solution, the same as when we ran it without search.
-
-Here is what happens. `krun` can only explore as much of the transition
-system associated to a program as `kompile` allowed the generated language
-model to yield. Since most of the K users are interested in language models
-that execute efficiently, that is, in faster interpreters for the defined
-languages, by default `kompile` optimizes the generated language model for
-execution. In particular, it inserts no backtracking markers, which `krun`
-uses when called with the `--search` option in order to systematically generate
-the entire transition system associated to a program. This is why `krun`
-showed us only one solution when run with the `--search` option on `div.imp`.
```

Comment:
Does `--search` still only return one solution here, or does it actually show all the different interleavings nowadays?

If it only returns one solution, I think we still need to include a good chunk of the prose here to explain why the other solutions are missing (as well as explain whatever the modern alternative to `--transition` is).

Reply:
I tested it and indeed `--search` returns all solutions now. I believe this was written before the search feature in the llvm backend was a thing.